test/onnx/onnx_import.in.cpp

```diff
@@ -706,61 +706,144 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_matmul)
-               3.63439123e-19f,
-               9.87929963e-19f,
-               2.68547207e-18f}},
-
-             {{7.29986992e-18f, 1.98431037e-17f, 5.39391483e-17f, 1.46621807e-16f, 3.98559393e-16f},
-              {1.08339676e-15f, 2.94497771e-15f, 8.00527940e-15f, 2.17606055e-14f, 5.91514586e-14f},
-              {1.60790335e-13f, 4.37073446e-13f, 1.18808881e-12f, 3.22956021e-12f, 8.77885484e-12f},
-              {2.38634016e-11f,
-               6.48674509e-11f,
-               1.76328013e-10f,
-               4.79309234e-10f,
-               1.30289758e-09f}},
-
-             {{3.54164282e-09f, 9.62718331e-09f, 2.61693974e-08f, 7.11357975e-08f, 1.93367146e-07f},
-              {5.25626399e-07f, 1.42880069e-06f, 3.88388295e-06f, 1.05574884e-05f, 2.86982290e-05f},
-              {7.80098743e-05f, 2.12052824e-04f, 5.76419338e-04f, 1.56687021e-03f, 4.25919482e-03f},
-              {1.15776919e-02f,
-               3.14714295e-02f,
-               8.55482149e-02f,
-               2.32544158e-01f,
-               6.32120559e-01f}}})
-            .get_vector();
+        0.98721677,  -0.05100663, -1.21178917, -0.17530157, 1.40051805,  -0.13259761, -1.14313018,
+        0.2673723,   -0.87996154, 1.29053106,  1.55,        0.8396538,   1.20729817,  0.23727845,
+        -0.89113606, -1.70909842, 0.26460363,  -0.70566808, 2.383518,    1.07024615,
+
+        -1.21722605, 0.82919357,  0.55765697,  0.12657686,  0.63432172,  0.75425957,  -2.43721014,
+        -1.24478184, 2.65316853,  1.19509542,  -0.95523998, 0.5149006,   -0.01151649, 0.68327026,
+        -0.4589638,  -0.46554745, 0.21055324,  0.39266729,  2.05098086,  1.83207919};
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_softmax_axis_0)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/softmax_axis_0.prototxt"));
 
     auto test_case = ngraph::test::NgraphTestCase(function, "${BACKEND_NAME}");
-    test_case.add_multiple_inputs(inputs);
-    test_case.add_expected_output(expected_output);
-    test_case.run();
+    test_case.add_input<float>(SOFTMAX_INPUT);
+
+    test_case.add_expected_output<float>(
+        {0.09683057, 0.00369363, 0.01394559, 0.00329012, 0.00234823, 0.00757665, 0.02449322,
+         0.02019284, 0.04985249, 0.00592694, 0.00279593, 0.04505148, 0.00641108, 0.00458466,
+         0.00348007, 0.00172928, 0.00330577, 0.01093237, 0.01554086, 0.10351497,
+
+         0.01648154, 0.00583583, 0.00182802, 0.00515374, 0.02491679, 0.00537859, 0.00195794,
+         0.00802367, 0.00254737, 0.0223216,  0.02893419, 0.0142204,  0.02053893, 0.00778581,
+         0.00251907, 0.00111174, 0.00800149, 0.0030324,  0.06658917, 0.0179084,
+
+         0.00181811, 0.01407243, 0.01072611, 0.0069699,  0.01158077, 0.01305647, 0.00053677,
+         0.0017687,  0.08719896, 0.02028982, 0.00236265, 0.01027717, 0.0060709,  0.01216173,
+         0.00388087, 0.00385541, 0.00758048, 0.00909469, 0.04775123, 0.03836337});
+
+    test_case.run(6);
```

Comment:
Check if using `Softmax` operator for cases >2d gives better accuracy.

Reply:
Unfortunately that didn't help. I have to leave those custom tolerance values for those 3 tests.